README.md

```diff
@@ -11,10 +11,27 @@ $ npm install -g truffle-checkout
 ## Usage
 
 ```
-$ tc [optional selectors]
+# Download, checkout, link, and `npm install` all of the truffle modules
+$ tc init
+
+# Checkout :org/:branch in all modules; useful for reviewing cross-cutting changes
+$ tc switch :org :branch
+
+# List all modules and their status
+$ tc list
+
+# Build symlinks between all packages. Run this in case of repair.
+$ tc link
+```
```

Comment:
Just a note - @tcoulter had originally wanted to avoid the use of subcommands, although I think they buy us a simpler implementation (as seen by your work!). Maybe we can fall-back on undefined commands, defaulting to `use`?